### DISCUSSION TITLE
MLFlow for commercial use, licensing considerations?

### DISCUSSION THREAD

**Discussion by KansaiUser** [mlflow/mlflow#6891] [Q&A] [answered]
I am starting to learn MLFlow and so far I like it. 
I am hoping to use it for my own projects, but there is a question lingering on my head

I have come to know that we are not able to use Anaconda for commercial use anymore without a paid license. 
In my company we have stopped using Anaconda.

I see that MLFlow uses docker and anacaonda frequently. My question is, can we use MLFlow in a company? 

Thanks

**Comment by SayakMukhopadhyay** [comment, marked as answer]
MLFlow itself is under Apache license so one should be able to use without commercial restrictions (apache license has some other conditions though which should be pretty easy to comply with). As for the dependency with Anaconda, check this note out https://github.com/mlflow/mlflow/blob/master/docs/source/models.rst#additional-logged-files. Seems like MLFlow now uses conda-forge which is the MIT licensed version.

**Reply by JHibbard** [reply]
This is correct, if you don't use MLflow integrations requiring a software license from third party tooling, e.g. anaconda, you should be good to go 👍
